js/overrides.js

```diff
@@ -1,61 +1,43 @@
-  'check region in frame hidden under top bar fully with scroll stitching': {skip: true},
-  'check region by selector in frame fully on firefox legacy': { skip: true },
-  // fails due to wrong calculation/rounding on other sdks
-  'appium android check window': {skipEmit: true},
-  'appium android check region with ignore region': {skipEmit: true},
-  // fails due to wrong screenshot size on other sdks
-  'should send region by selector in padded page': {skipEmit: true}
+    /**
+     *  Fails because of the difference between SCREEN SIZE and LAYOUT VIEWPORT SIZE.
+     *  We should use screen.width/screen.height for cropping/scaling and innerWidth/innerHeight to understand element location related to the viewport
+     */
+    'check region in frame hidden under top bar fully with css stitching': {skip: true},
+    'check region in frame hidden under top bar fully with scroll stitching': {skip: true},
+    // Rendered image isn't fully
+    'check frame fully with vg': {skip: true},
+    /// Updating baselines to JS
+    // Window
+    'check window fully on android chrome emulator on desktop page': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check window fully on page with sticky header with scroll stitching': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check window two times with vg classic': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check window with vg classic': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check window with vg': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check window fully and frame in frame fully with css stitching': {config: {branchName: 'universal-sdk'}}, // was v2
+    'check window fully and frame in frame fully with scroll stitching': {config: {branchName: 'universal-sdk'}}, // was v2
+    'check window fully and frame in frame fully with vg': {config: {branchName: 'universal-sdk'}}, // was v2
+    'check window fully with custom scroll root with css stitching': {config: {branchName: 'universal-sdk'}}, // was v2
+    // Region
+    'check region by selector fully on page with sticky header with scroll stitching': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check region by coordinates in frame with css stitching': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check region by coordinates in frame with scroll stitching': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check overflowed region by coordinates with css stitching': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check overflowed region by coordinates with scroll stitching': {config: {branchName: 'universal-sdk'}}, // was skipped
+    'check regions by coordinates in overflowed frame with scroll stitching': {config: {branchName: 'universal-sdk'}}, // was next
+    'check region by selector with vg classic': {config: {branchName: 'universal-sdk'}}, // was no-fully-by-default
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
    "check region by selector with vg classic": {config: {branchName: 'universal-sdk'}}, // was no-fully-by-default
```